parcellab/monolith/values.yaml

```diff
@@ -72,26 +72,9 @@ extraIngresses: []
 #     - chart-example.local
 
 ##
-## Cronjob
+## Cronjobs
 ##
 
-# Only for the defaults
-cronjob:
-  enabled: true
-  concurrencyPolicy: "Allow"
-  failedJobsHistoryLimit: 1
-  schedule: "* * * * *"
-  successfulJobsHistoryLimit: 3
-  suspend: false
-  job:
-    activeDeadlineSeconds: 900 # 15 minutes
-    backoffLimit: 2
-
-  #   command:
-  #   - /bin/sh
-  #   - -c
-  #   - date; echo Hello from the Kubernetes cluster
-
```

Comment:
If you delete this, the defaults won't be set and it will fail. We could of course set such defaults in the `common` lib as an alternative if we want to avoid this config.

However, it should be `cronjob.enabled: false`